Signal.xcodeproj/project.pbxproj

```diff
@@ -2449,7 +2453,7 @@
 					"\"$(SRCROOT)/Libraries\"/**",
 				);
 				INFOPLIST_FILE = "$(SRCROOT)/Signal/Signal-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
Shall we also bump:

* https://github.com/WhisperSystems/SignalServiceKit/blob/master/SignalServiceKit.podspec#L24
* https://github.com/WhisperSystems/SignalProtocolKit/blob/master/Podfile#L1

Strangely, this one is already bumped!

* https://github.com/WhisperSystems/Signal-iOS/blob/master/Podfile#L1